charts/spire/charts/spire-server/templates/bundle-configmap.yaml

```
@@ -4,3 +4,7 @@ kind: ConfigMap
 metadata:
   name: {{ include "spire-lib.bundle-configmap" . }}
   namespace: {{ .Values.notifier.k8sbundle.namespace | default $namespace }}
+  {{- with .Values.configMap.annotations }}
```

These annotations are specific to the bundleConfigmap.

```suggestion
  {{- with .Values.bundleConfigMap.annotations }}
```